### DISCUSSION TITLE
How to get BSSId http request

### DISCUSSION THREAD

**Discussion by eightbit88** [arendst/Tasmota#20105] [Q&A] [answered]
Hi all, I'm trying to get the mac address of the AP my ESP32 is connected to. I can get a lot of other data via
http://10.10.10.16/cm?cmnd=status%200    and the 1 - 10 e.g. http://10.10.10.16/cm?cmnd=status%205   (5)
However {{payload.Status.BSSId}}  is not returning anything neither is {{payload.StatusNet.BSSId}}

This is for a tracking application and I need the MAC of the AP the ESP is connected to. I could use the name but this would not be very reliable as there is a possible user error.

Any help would be greatly appreciated.

Regards

**Comment by barbudor** [comment, marked as answer]
Have you looked with your own eyes where the BSSId could be located?
It looks in StatusSTS (response to Status11)

**Comment by eightbit88** [comment]
Thank you for your reply, I did some searches on the API and could not find
anything. Would you be so kind to let me know where this information is
located.
I’m only starting to learn about Tasmota, my sincere apologies if I ask
simple question.

**Reply by barbudor** [reply]
I have no idea how I could be more explicit than what i'v already written : in the StatusSTS object that is the response to command Status11

**Comment by eightbit88** [comment]
Yes, I fully understood, and your answer could have not be clearer to my
question, thank you. My question on the other hand was not that clear, so
let me try again.
Where can I find documentation relating to the API commands and responses?
Many thanks.

**Reply by barbudor** [reply]
https://tasmota.github.io/docs/Commands

responses are not documented. you have to try and see
